src/Faithlife.PackageDiffTool/LocalPackageHelper.cs

```diff
@@ -96,9 +99,24 @@ public async Task<PackageReaderBase> GetLocalPackageAsync(string packagePath, Ca
 			return new PackageFolderReader(packageFolder);
 		}
 
+		public void Dispose()
+		{
+			foreach (var directory in m_tempDirectories)
+			{
+				try
+				{
+					Directory.Delete(directory, true);
+				}
+				catch (IOException)
```

Comment:
FWIW we catch more exception types here: https://github.com/Faithlife/FaithlifeUtility/blob/eadb97bb7c2076ce61e4d8e4f966133479706564/src/Faithlife.Utility/DirectoryUtility.cs#L24-L34